docs/editors/vim.md

```diff
@@ -17,7 +16,8 @@ which will provide the most complete implementation of LSP and Metals specific h
 ## Installing Vim
 
 The coc.nvim plugin requires either **Vim >= 8.1** or **Neovim >= 0.3.1**. Make
-sure you have the correct version installed.
+sure you have the correct version installed. While it works with both Vim and Neovim, the
+experience is a bit smoother with Neovim.
```

Comment:
I originally wasn't going to mention anything about a preference for Neovim, but after spending some time with the new Vim 8.2, I feel pretty confident in saying that Neovim is a smoother experience. It's snappier with larger codebases for one, but another thing leading me to say this is that code actions seem sort of unusable at the moment with Vim 8.2 and coc.nvim (see https://github.com/neoclide/coc.nvim/issues/1404)